core/src/banking_stage/consumer.rs

```diff
@@ -73,13 +74,13 @@ pub struct ExecuteAndCommitTransactionsOutput {
 #[derive(Debug, Default, PartialEq)]
 pub struct ExecuteAndCommitTransactionsCounts {
     // Total number of transactions that were passed as candidates for execution
-    pub(crate) attempted_execution_count: u64,
-    // The number of transactions of that were executed. See description of in `ProcessTransactionsSummary`
+    pub(crate) attempted_processing_count: u64,
+    // The number of transactions of that were processed. See description of in `ProcessTransactionsSummary`
     // for possible outcomes of execution.
-    pub(crate) executed_count: u64,
-    // Total number of the executed transactions that returned success/not
+    pub(crate) processed_count: u64,
+    // Total number of the processed transactions that returned success/not
```

Comment:
Can a processed transaction have failed? I think this parameter counts the number of transaction that succeeded (were executed).

Reply:
> Can a processed transaction have failed?

Yes a processed transaction could have had "failure" result meaning that we definitely validated the fee payer but may or may not have executed it successfully. 

> I think this parameter counts the number of transaction that succeeded (were executed)

This parameter used to count the number of transactions that were executed (whether the execution result was success or failure, is irrelevant). Now it counts the number of transactions that were "processed" (meaning valid for including in a block) whether they were executed or not.